modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -29,6 +29,9 @@ class OrtResultAnalyzer : ManualAnalyzer() {
         private val LOGGER = LoggerFactory.getLogger(OrtResultAnalyzer::class.java)
     }
 
+    private val filterExcludedArtifactsProperty = "filter.excluded"
```

Comment:
I'm always a bit confused about whether "to filter" means "to keep something" or "to get rid of something". If we follow the logic of [Kotlin's filter](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/filter.html), it "returns a list containing only elements matching the given predicate", so here "to filter" means "to keep the filtrate". Further following that logic means that setting `filter.excluded = true` should *keep* (only) the excludes, which is not what we want here.

TL;DR, I propose to rename this property to "omit.excludes" to 1) avoid this confusion and 2) align with ORT's naming of the parameter. Further, I'd argue we could / should set the default to `true`, because in the context of creating reports, I'd say you always want to omit excluded packages.

Reply:
Good point, followed your proposal in the new commit